### DISCUSSION TITLE
Job schedulers specification

### DISCUSSION THREAD

**Discussion by simaoribeiro** [open-telemetry/opentelemetry-specification#2170] [Q&A] [answered]
Hi there,

my team is working on the transition from OpenTracing to OpenTelemetry standards within the company and we're in doubt about what should be the guidelines regarding span names and attributes for job schedulers (specifically, we're using Hangfire),

That specification is something that is being worked on currently? If not, is there someone that wants to share their current customized implementation?

**Comment by pyohannes** [comment, marked as answer]
As far as I know, there's nobody currently actively working on a specification for this.

There's a group working on stabilizing messaging semantic conventions, however, job schedulers like Hangfire were defined as [out-of-scope for that effort](https://github.com/open-telemetry/oteps/blob/main/text/trace/0173-messaging-semantic-conventions.md#asynchronous-message-passing-in-the-wider-sense).

This will likely be a separate semantic convention document. [Some work on this was started](https://github.com/open-telemetry/opentelemetry-specification/pull/1582), but was abandoned. If you want to start working on this, the abandoned PR might be a good starting point.

**Reply by simaoribeiro** [reply]
Thank you @pyohannes . I will take a look
